CHANGELOG.rst

```diff
@@ -19,6 +19,7 @@ CHANGELOG
 * Fix tabular tables inline fields
 * Fix add another button size on advanced page settings
 * Fix tagged items delete label position
+* Fix login password field
```

Comment:
there is something off, 1.0.6 has been released

can you merge against master?